vaadin-ordered-layout-flow-parent/vaadin-ordered-layout-flow-integration-tests/src/main/java/com/vaadin/flow/component/orderedlayout/it/ScrollerPage.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2000-2024 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.orderedlayout.it;
+
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.NativeButton;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.Scroller;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.router.Route;
+
+/**
+ * View for the ordered layouts {@link HorizontalLayout}, {@link VerticalLayout}
+ * and {@link Scroller}.
+ *
+ * @author Vaadin Ltd
+ */
+@Route("vaadin-ordered-layout/scroller")
+public class ScrollerPage extends Div {
```

Comment:
The scroller on the test page is not scrollable

![Screenshot 2024-07-24 at 11 00 13](https://github.com/user-attachments/assets/37889a76-ed3e-4891-915b-679f83b7dd7e)

Reply:
I mistakenly removed the width and height from the view while cleaning the file.

I now added a test that makes sure that when the content is at the bottom, it's not at the top. This assert shouldn't hold if the test is scrollable.